### PR TITLE
Add Summit-XL Steel 

### PR DESCRIPTION
**Pull request by mael25**
**Description**
Adds the Robotnik Summit-XL Steel robot
**This PR replaces #2969 which targeted the `master` branch**

**Tasks**
  - [x] Create Icons
  - [x] Documentation Summit-XL Steel
  - [x] Better appearance & documentation SickS300 (lidar-sensors.md)
  - [x] Update robots Makefile
  - [x] Update Changelog
  - [x] Improve appearance base & Redo pictures/icons
  - [x] Add web component
  - [x] Fix `contactProperties` issue
  - [x] Add missing sensors

.

### CODE REVIEW

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
The plastic crate that is on the robotino plateform is a bit _in_ the conveyor belt and so it cannot move.

File: docs/guide/summit-xl-steel.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+## Robotnik's Summit-XL Steel
+
+%robot summit-xl-steel images/robots/summit_xl_steel/model.thumbnail.png
+
+Designed by Robotnik, [Summit-XL Steel](summit-xl-steel.md) is a four-wheeled (mecanum) robot.
```
[the remark this answers is not in the document]
Comment (mael25) [reply docs/guide/summit-xl-steel.md]: You're right, I have replaced it by the link to the webpage on the constructor website, as it is done for several other robots currently.

**Comment by mael25** [comment]
@BenjaminDeleze I have slightly offseted the `PlasticCrate` so that it moves on the conveyor belt. I have also added the missing sensors (`GPS` and RGBD camera, implemented as a `Camera` and a `RangeFinder`)

**Review by BenjaminDeleze — APPROVED** [review]
Thank you!

**Pull request merged**